### ISSUE TITLE
Replace code of conduct

### ISSUE THREAD

**Issue by BigHerzog** [Sprytile/Sprytile#113]
I agree with a lot of criticisms of it being too vaguely worded (and maybe a bit too wordy :laughing: ). I think something like the ruby community conduct guideline is much more concise and clear, and still has the same spirit as the CC, disallowing harassment and bigoted and personal attacks and remarks, as well as promoting tolerance and respect (though there is technically leeway for some swearing, but I don't think that's a problem hahaha):

> This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to the Sprytile community. It applies to all “collaborative space”, which is defined as community communications channels (such as mailing lists, submitted patches, commit comments, etc.).
> 
> - Participants will be tolerant of opposing views.
> - Participants must ensure that their language and actions are free of personal attacks and disparaging personal remarks.
> - When interpreting the words and actions of others, participants should always assume good intentions.
> - Behaviour which can be reasonably considered harassment will not be tolerated.
> 

What do you guys think? Although this isn't the only one, and maybe we could work together to come up with a custom one or something, but that might be too ambitious :stuck_out_tongue:

**Comment by BigHerzog** [comment]
It'd be cool if you responded instead of just using the emojis :(

**Comment by unknowndevice** [comment]
it's already been discussed and it's been stated that the owner is not interested in changing the CoC. Creating an account just to try and bring this up again doesn't seem like you are bringing this issue up in good faith.

**Comment by BigHerzog** [comment]
>it's already been discussed and it's been stated that the owner is not interested in changing the CoC. 

I wasn't aware of this, sorry. I was just hoping to bring up the various issues with it, such as the ambiguity and lack of definitions. Can you link me the issue?

>Creating an account just to try and bring this up again doesn't seem like you are bringing this issue up in good faith.

I didn't just make an account to post issues, it's just that I am kind of privacy conscious so I frequently make new ones.

**Comment by BigHerzog** [comment]
Hmm, looking at the discord server, the dev seems to be an actual socialist lmao. I guess adding the CoC is a tool to keep away developers who disagree with him. Sad to see such a cool project managed by such a fascist :(
